vitest.config.ts

```diff
@@ -60,7 +60,7 @@ export default defineConfig(
 			environment: 'jsdom',
 			globals: true,
 			watch: false,
-			silent: true,
+			silent: false,
```

Comment:
I suppose this is necessary for the error string check

Reply:
Thx! Leftover, reverted but, provided in another PR #2684